exchange_calendars/exchange_calendar_xtai.py

```diff
@@ -169,14 +193,14 @@ def bridge_fri(dt: datetime.datetime) -> datetime.datetime | None:
     month=4,
     day=4,
     start_date="2011",
-    observance=bridge_mon,
+    observance=partial_bridge_mon,
```

Comment:
Could you provide a reference (within the PR comments is fine) confirming that this extra Monday is not applied from 2024.